src/object/srv_obj.c

```diff
@@ -2007,7 +2017,7 @@ obj_capa_check(struct ds_cont_hdl *coh, bool is_write, bool is_agg_migrate)
 		return -DER_NO_PERM;
 	}
 
-	if (!is_agg_migrate && coh->sch_cont && coh->sch_cont->sc_rw_disabled) {
+	if (!is_agg_migrate && coh->sch_cont && !is_cont_rw(is_write, coh->sch_cont)) {
```

Comment:
hmm, I thought you need add check ?

if (is_write && !coh->sch_cont->sc_write_disabled)
              return -DER_NO_PERM;